test/app/components/photo_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class PhotoComponent < ViewComponent::Base
```

Comment:
Might we name this to correspond with the feature it is meant to exercise? The actual function / content of the component is excellent 👌 

```suggestion
class CollectionCounterComponent < ViewComponent::Base
```